lib/route53.js

```diff
@@ -7,6 +7,52 @@ const Promise = require("bluebird");
 const CustomError = require("./errors")
 const DNSHandler = require("./base")
 
+
+function changeRequestMethod(params, method) {
+	params.ChangeBatch.Changes[0].Action = method;
+	return params;
+}
+
+function generateParams(hostedZoneId, domain, opts) {
+	let params = {
```

Comment:
Minor: consider using const as much as possible for any variables that don't change.